src/pages/upload/index.tsx

```diff
@@ -74,18 +121,38 @@ const Upload: React.FC = () => {
         onError={(message) => console.log(message)}
         onFinish={(signResult, file) => {
           const fileUrl = signResult.signedUrl.split('?')[0]
-          console.log(fileUrl, signResult.publicUrl, file)
+          setLessonMetadata((prevState) => [
```

Comment:
Each file upload formed a clojure on the empty form state. And so when the state was updated only the latest file uploaded would be saved. 

the useState hook allows you to access the previous state in a callback function. By spreading the previous state we are able to prevent conflicts.